packages/notte-browser/src/notte_browser/session.py

```diff
@@ -24,6 +24,8 @@
 from notte_core.llms.engine import LlmModel
 from notte_core.llms.service import LLMService
 from notte_core.utils.webp_replay import ScreenshotReplay, WebpReplay
+from notte_integrations.pdf.docling.serve import DoclingPDFHandler
```

Comment:
integrations should not be imported in notte. We don't want to create this dependency. You can import `BasePDFHandler` though and that should be enough.